### PR TITLE
TII-194 amendment; use new icon for content review service

### PR DESCRIPTION
**Pull request by bjones86**
Addressed the issue from https://github.com/sakaicontrib/sakai-turnitin/pull/1 @buckett brought up.

### CODE REVIEW

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm [hunk 1]
```diff
@@ -735,7 +735,9 @@ $(document).ready(function(){
 						#end
 					</h3>
 					#if ($assignment.getContent().getAllowReviewService())
-						<p class="instruction"><img alt="$reviewIndicator" title="$reviewIndicator" src="/library/image/silk/rosette.png" /> $reviewIndicator</p>
+						<p class="instruction">
+							<i alt="$reviewIndicator" title="$reviewIndicator" class="fa fa-certificate reviewStyle">$reviewIndicator</i>
```
Comment (buckett) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: I think it should actually be:
```
    <i alt="$reviewIndicator" title="$reviewIndicator" class="fa fa-certificate reviewStyle"></i>
```
but actually there are lots of places that still have the silk icon so maybe we should just extract out a macro and update all the locations. The macro then makes it easier for support 10/11.

Comment (bjones86) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: I don't see any other references to the rosette.png silk icon, but I do see some references to other silk icons in the code base. I'm not sure how we would address that with a macro though.

Comment (buckett) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: Sorry I meant other references to the review indicator that will want to be the silk icon for 10 and the font awesome character for 11.

Comment (bjones86) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: I'm not sure we need to worry about 10 in this code base. I was under the impression that we at Western would just be providing patches if anyone else were looking to run the LTI integration with 10.

**Pull request merged**